frontend/src/components/common/selector/selector.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  FC,
+  FormControl,
+  FormControlErrors,
+  SelectorOptions,
+} from 'common/types/types';
+import { ErrorMessage } from 'components/common/common';
+import { useFormControl } from 'hooks/hooks';
+
+import styles from './styles.module.scss';
+
+type Props = {
+  control: FormControl;
+  errors: FormControlErrors;
+  name: string;
+  options: SelectorOptions[];
+  setValue?: (name: string, value: string) => void;
+  value?: string;
+  label: string;
+};
+
+const Selector: FC<Props> = ({ name, control, errors, label, options }) => {
```

Comment:
```suggestion
const Select: FC<Props> = ({ name, control, errors, label, options }) => {
```
don't forget about file name